### DISCUSSION TITLE
Is there a way to truncate the table?

### DISCUSSION THREAD

**Discussion by devsarvesh92** [piccolo-orm/piccolo#1340] [Q&A]

**Comment by dantownsend** [comment]
Piccolo doesn't have a truncate command built in at the moment - you can use delete:

```python
await MyTable.delete(force=True)
```

Or raw SQL to access the truncate command:

```python
await MyTable.raw('TRUNCATE my_table')
```

**Comment by dantownsend** [comment]
I'll probably add this when I get time, as I do think would be nice.

Sometimes along the lines of this:

https://github.com/piccolo-orm/piccolo/blob/9d1ab2d492c26a677d689823f85d12303f548bfa/piccolo/table.py#L1449

**Comment by sarvesh-deserve** [comment]
Thanks @dantownsend
